### PR TITLE
smartattributes: fix bug

### PR DESCRIPTION
**Pull request by tvdijen**
Fixes https://github.com/simplesamlphp/simplesamlphp/issues/841

### CODE REVIEW

File: modules/smartattributes/lib/Auth/Process/SmartID.php [hunk 1]
```diff
@@ -1,116 +1,123 @@
...
+     * eduPersonTargetedID, or there will be no way to see its origin any more.
+     */
+    private $_candidates = array(
+        'eduPersonTargetedID',
+        'eduPersonPrincipalName',
+        'openid',
+        'facebook_targetedID',
+        'twitter_targetedID',
+        'windowslive_targetedID',
+        'myspace_targetedID',
```
Comment (jaimeperez) [inline modules/smartattributes/lib/Auth/Process/SmartID.php]: Is this related?

Comment (tvdijen) [reply modules/smartattributes/lib/Auth/Process/SmartID.php]: It's just whitespace fixes you're looking at.. This code is not new.

Comment (jaimeperez) [reply modules/smartattributes/lib/Auth/Process/SmartID.php]: No, I was skipping whitespace changes explicitly, and then the `myspace_targetedID` appeared as new. That's why I was wondering whether it was related to the bugfix or not... 🤔

Comment (thijskh) [reply modules/smartattributes/lib/Auth/Process/SmartID.php]: I guess a merge error since the line has been removed in 852ee7e32a75aac93263e6b465567eedfd0f02d5.

Comment (jaimeperez) [reply modules/smartattributes/lib/Auth/Process/SmartID.php]: In that case, I think it's a good idea to remove any leftovers. Other than that: 👍

Comment (tvdijen) [reply modules/smartattributes/lib/Auth/Process/SmartID.php]: I'll fix it later today
